src/components/TabBar/TabBar.jsx

```diff
@@ -47,14 +48,21 @@ const StyledSection = styled.section`
   }
 `;
 
+const filteredURLSearchParams = () => {
+  const PARAMS_TO_IGNORE = [constants.PAGINATION_PAGE_NUMBER, constants.PAGINATION_PAGE_SIZE];
+  const params = querystring.parse(window.location.search.substring(1));
+  PARAMS_TO_IGNORE.forEach(key => delete params[key]);
+  return '?' + querystring.encode(params);
```

Comment:
String interpolation?

Reply:
Yeah, it looks like this was causing the build to fail, _should_ be fixed now